### ISSUE TITLE
[COGNITION]: Screen reader text for timestamps should match visual timestamps

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#25182]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement

As a screen reader user, I want the timestamp that gets read out to match what's printed on the screen. Hearing a date time format but seeing text like "5 minutes ago" could be confusing.

## Details

We should not be changing the format of the screen reader timestamp text from the visual text. It creates a cognitive challenge trying to reconcile the different data formats and "find" what's being spoken. Screenshot attached below.

## Acceptance Criteria

- [ ] Screen reader timestamps match visual text

## Environment

* https://staging.va.gov/virtual-agent/

## WCAG or Vendor Guidance (optional)

* https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html

## Screenshots or Trace Logs

![Screen Shot 2021-05-24 at 2 57 28 PM](https://user-images.githubusercontent.com/934879/119409925-a5165380-bcad-11eb-938e-778e7f381743.png)

**Comment by ianpsant** [comment]
Custom - Within our control and we are able to change. We can track this accessibility issue and address if PVA development moves forward towards production.

**Comment by joshkimux** [comment]
@ianpsant Hi Ian-- we typically recommend resolving serious accessibility issues (defect-2)s within 1-2 sprints. Was this issue blocked by any chance?

If you (or if there's a new team/pm) need help, feel free to reach out to the CAIA team at #accessibility-help on slack ❤️

**Comment by ianpsant** [comment]
At the time of our launch we were asked to address defect-1's and never got back round to this. We can look at this again, perhaps Microsoft addressed it on their end. We'll also be going through another collaboration cycle soon when we move the chatbot to a more prominent place on VA.gov later this year.
